cardano-crypto-class/cardano-crypto-class.cabal

```diff
@@ -101,6 +100,8 @@ test-suite test-crypto
   default-language:    Haskell2010
 
   ghc-options:         -Weverything
+                       -Wno-orphans
+                       -Wno-incomplete-uni-patterns
```

Comment:
That is ok for a test suite, but I would be nervous about having that in production code.

Reply:
Yes, that's why it's only in the test suite :-)

I can move it back to the modules individually. Or we can switch the testsuite from -Weverything to -Wall.

Reply:
I already moved it, to fix a stack CI build error.